### PR TITLE
Add paragraph styles to main extension popup

### LINKED ISSUE #38: Add CC vocabulary paragraph styles to main extension popup
There is some initial information when the extension opens. Add CC vocabulary styles to it.

**How to do it**
Note - CC-vocabulary is already linked into the project, therefore you just need to add the appropriate classes and mimic the markup(if different) to change the styles.

Head over to the [Paragraph section](https://opensource.creativecommons.org/cc-vocabulary/styleguide/#/Elements/Paragraph) in CC-Vocabulary docs. Inspect the paragraph element darker shaded blue color in dev-tools and copy its classes.
 
Use the copied classes on the [paragraph element](https://opensource.creativecommons.org/cc-vocabulary/styleguide/#/Elements/Paragraph) of the extension popup. Note that the current classes in the element are of use, therefore wrap the paragraph in a div tag with those classes.

### PR DESCRIPTION
**Pull request by sharmaadityaHQ**
Fixes #38 

**Description**
Added CC vocabulary paragraph styles to main extension popup.

**Checklist:**
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [ ] I added tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

### CODE REVIEW

**Comment by makkoncept** [comment]
@sharmaaditya570191 please paste the screenshots of the changes

**Comment by sharmaaditya570191** [comment]
Hi @makkoncept .I would add the screenshots in a while. Thanks.

**Comment by sharmaaditya570191** [comment]
**Before**
Screenshot from 2019-12-04 19-59-56

**After**
Screenshot from 2019-12-04 20-01-48

**Comment by sharmaaditya570191** [comment]
Hi @makkoncept . Please suggest if something else is to be changed here.

**Comment by ritesh-pandey** [comment]
@sharmaaditya570191 I think we will have to make changes [here](https://github.com/creativecommons/ccsearch-browser-extension/blob/master/src/utils.js#L34) as well.

File: src/popup/popup.html [hunk 1]
```diff
@@ -65,7 +65,7 @@
         </section>
         <span id="error-message"></span>
         <section class="section-content section-content--primary">
-          <div class="row">
+          <div class="row vocab paragraph inherit-colored">
```
Comment (makkoncept) [inline src/popup/popup.html]: Add these classes directly to the paragraph tag.

**Review by makkoncept — CHANGES_REQUESTED** [review]
I have added a comment. Also, @ritesh-pandey pointed out a valid point. Please make the necessary changes.

**Comment by sharmaaditya570191** [comment]
**After Changes**

Screenshot from 2019-12-06 19-25-03

**Pull request merged**